clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go

```diff
@@ -463,13 +468,17 @@ type objectChecker struct {
 	stopCh    chan struct{}
 	status    objectStatus
 	err       error
+	// key of the object being checked. In the current implementation it's a namespaced name, but it
+	// may change in the future.
+	key string
 }
 
-func newObjectCheker() *objectChecker {
+func newObjectChecker(key string) *objectChecker {
 	return &objectChecker{
-		stopCh:    make(chan struct{}),
-		isRunning: true,
-		status:    unknown,
+		stopCh:         make(chan struct{}),
+		isRunning:      true,
+		status:         unknown,
+		key: key,
```

Comment:
this will fail gofmt

Reply:
Doh, always forgetting about gofmt. 
Done.